### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by patrinoua**
### Description

Some more small changes to the formating:
- [x] Brief reminder text that's easy to read.
- [x] Order by what occurs more often in development.
- [x] Add ticket number on the PR name so it's visible in the list of open PRs.

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-- [ ] Screenshots/videos of my changes.
-- [ ] Tested on **iOS** and **Android**.
-- [ ] Tests/stories for my changes.
+<!-- 💡 MOPLAT warmly welcomes any feedback about the list or how it impacts your workflow. -->
+
+- [ ] Tested on **iOS** and **Android**
+- [ ] Screenshots and videos 
+- [ ] Added Tests and Stories
+- [ ] App state migration
+- [ ] Feature flag
```
Comment (MounirDhahri) [inline docs/pull_request_template.md:20]: @pvinis this is what I meant for example with making the template also clear for open-source contributors with my comments here https://github.com/artsy/eigen/pull/6547#issuecomment-1090241129 . `Feature flag` is **kind of** clear to artsy engineers but for someone not from Artsy, they would find a hard time to understand if this is actually a feature flag pr, a change to feature flags, changes hidden by feature flags. And I do believe in providing valuable context https://artsy.github.io/blog/2020/08/11/improve-pull-requests-by-including-valuable-context/

**Review by MounirDhahri — APPROVED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (rajsam003) [reply docs/pull_request_template.md:20]: It looks clean to me without too much content. 
Maybe we can attach a notion page link explaining each point in detail with examples.

**Comment by MounirDhahri** [comment]
🙌

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
+  `type(PROJECT-XXXX): what changed`
 -->
 
 <!-- Jira ticket in square brackets like [PROJECT-XXXX] -->
 This PR resolves []
 
 ### Description
 
-<!-- Some info, Implementation, Before/After shots, follow-up work, etc -->
+<!-- Info, implementation, how to get there, before & after screenshots & videos, follow-up work, etc -->
```
Comment (MounirDhahri) [inline docs/pull_request_template.md:10]: **praise:** 👏

**Review by rajsam003 — APPROVED** [review]
Angela The Refiner 👩‍🔧

**Pull request merged**